### PR TITLE
[serverless-apollo-contentful] Configure test coverage report to ignore all index.ts files

### LINKED ISSUE #1042: [serverless-apollo-contentful] Configure test coverage report to ignore all index.ts files
# Description
- test coverage report: ignore all index.ts files. These files are only for modules (re-export what is exported from files in folders)

### PR DESCRIPTION
**Pull request by iansamz**
## Type of change

- [ ] Documentation change
- [ ] Bug fix

## Checklist

- [ ] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [ ] This starter kit has been approved by the maintainers
- [ ] Changes for this new starter kit are being pushed to an integration branch instead of main
- [ ] All dependencies are version locked
- [ ] This fix resolves #1042
- [ ] I have verified the fix works and introduces no further errors

### CODE REVIEW

File: starters/serverless-apollo-contentful/jest.config.ts [hunk 1]
```diff
@@ -7,7 +7,12 @@ const config: JestConfigWithTsJest = {
 	collectCoverageFrom: ['src/**/*.ts'],
 	coverageReporters: ['html', 'json', 'lcov', 'text', 'clover'],
 	coverageDirectory: 'coverage',
-	coveragePathIgnorePatterns: ['/node_modules/', '/test/', '/mocks/'],
+	coveragePathIgnorePatterns: [
+		'/node_modules/',
+		'/test/',
+		'/mocks/',
+		'/index.ts',
```
Comment (mbicknese) [inline starters/serverless-apollo-contentful/jest.config.ts:14]: Does this include index files not in the root folder?

e.g., `/some/path/index.ts`?

Comment (iansamz) [reply starters/serverless-apollo-contentful/jest.config.ts:14]: It does ignore

Comment (iansamz) [reply starters/serverless-apollo-contentful/jest.config.ts:14]: It does. Here is the explanation: https://stackoverflow.com/a/64979756/6523474
Here is after: 
image

Here before: 

image

**Review by mbicknese — APPROVED** [review]
Ship it ⛵

**Pull request merged**